src/main/java/org/takes/facets/flash/RsFlash.java

```diff
@@ -76,7 +77,14 @@ public RsFlash(final String msg, final Level level) {
     public RsFlash(final String msg, final Level level, final String cookie) {
         super(
             new RsWithCookie(
-                cookie, msg,
+                cookie,
+                DatatypeConverter.printBase64Binary(
+                    new StringBuilder(level.getName())
+                        .append('/')
+                        .append(msg)
+                        .toString()
+                        .getBytes(Charset.defaultCharset())
```

Comment:
I also wanted to use UTF-8. But in many classes Take framework used default charset. I thought that there is reason for this.
I made #79 to clarify this